src/include/ipc4/base_fw.h

```diff
@@ -558,6 +558,21 @@ struct ipc4_system_time_info {
 	struct ipc4_system_time dsp_time;
 } __attribute__((packed, aligned(4)));
 
+struct ipc4_ext_system_time {
+	//! Lower DWORD of current system time value.
```

Comment:
align comments style to the rest in this file

Reply:
It is a good idea, the IPC4 interface requires _Doxygen & Breathe_ compatible documentation, but it should be introduced as a separate PR for the entire file.